poster/poster.tex

```diff
@@ -0,0 +1,428 @@
+		
+		\draw[->, thick] (of) -- (dem);
+		\draw[->, thick] (dem) -- (con);
+		
+		\end{tikzpicture}
+		
+	}
+\end{figure}
+Need for : \\ Demand Repsonsive Deployment Decisions
+\begin{figure}
+	\centering
+	\scalebox{0.80}{
+		\begin{tikzpicture}[    >=stealth,
+		node distance=3cm,
+		on grid,
+		align=center,
+		auto]
+		% Place nodes
+		\node [block] (rea) {{\LARGE \texttt{Reactor}}};
+		\node [cloud, below of=rea] (d1) {1. Demand: \mathbf{Fuel}};
+		\node [cloud, below of=d1] (s1) {2. Deploy \texttt{Enrichment} Facility \\ to meet fuel demand};
+		\node [cloud, below of=s1] (ff) {3. Deploy \texttt{Mills} to meet UF6 demand};
+		\node [cloud, below of=ff] (d2) {4. Deploy \texttt{Mines} to meet uranium demand};
+		
+		
+		\draw[->, thick] (rea) -- (d1);
+		\draw[->, thick] (d1) -- (s1);
+		\draw[->, thick] (s1) -- (ff);
+		\draw[->, thick] (ff) -- (d2);
+		\end{tikzpicture}
+		
+	}
+	\caption{Dynamic Deployment to meet fuel demand}
+\end{figure}
+
+
+\end{block}
+
+\end{column} % End of the first column
+
+\begin{column}{\sepwid}\end{column} % Empty spacer column
+
+\begin{column}{\twocolwid} % Begin a column which is two columns wide (column 2)
+
+\begin{columns}[t,totalwidth=\twocolwid] % Split up the two columns wide column
+
+\begin{column}{\onecolwid}\vspace{-.6in} % The first column within column 2 (column 2.1)
+
+%----------------------------------------------------------------------------------------
+%	MATERIALS
+%----------------------------------------------------------------------------------------
+
+\begin{block}{Methods}
+	A meta-review of previous \gls{NFC} gap analyses helped to identify the existingn
+	simulators and their capabilities. The strongest comparison of transitoin scenario
+	code capabilities is found in \cite{boucher_international_2010}, \cite{brown_identification_2016} and \cite{mccarthy_benchmark_2012}.
+	In \cite{carre_overview_2009} and \cite{hoffman_expanded_2016}, the ability of
+	individual simulators to conduct transition scenarios is addressed, but not the
+	flexibility and performance of their varying algorithms for this capability.
+	
+
+\end{block}
+
+%----------------------------------------------------------------------------------------
+
+\end{column} % End of column 2.1
+
+\begin{column}{\onecolwid}\vspace{-.6in} % The second column within column 2 (column 2.2)
+
+%----------------------------------------------------------------------------------------
+%	METHODS
+%----------------------------------------------------------------------------------------
+
+\begin{block}{Successful Applications}
+{\large 	\begin{itemize}
+		\item Lumber mills \cite{yanex_agent-based_2009}
+		\item coupling building efficiency with weather \cite{gonzalez_detailed_2002, kusiak_data-driven_2010}
+		\item airline industry \cite{shebalov_practical_2009}
+	\end{itemize}
+	}
+The success of these algorithms in other domains for similar classes 
```

Comment:
This text should appear immediately after the section heading, before the bulletted list. Also, perhaps phrase it "These algorithms have succeeded in other domains for similar classes of problems. Accordingly, their potential in nuclear fuel cycle analysis is promising."